pkg/machinehealthcheck/machinehealthcheck.go

```diff
@@ -46,10 +46,6 @@ var _ = Describe("[Feature:MachineHealthCheck] MachineHealthCheck", func() {
 		providerSpec := workers[0].Spec.Template.Spec.ProviderSpec.DeepCopy()
 		clusterName := workers[0].Spec.Template.Labels[framework.ClusterKey]
 
-		// TODO(bison): This should probably be appended with
-		// something other than a timestamp, e.g. a random string.
-		msName := fmt.Sprintf("e2e-mhc-%d", time.Now().Unix())
```

Comment:
i see that msName has moved to the outer scope, but i am wondering about this TODO note. do we need to have some sort random string on the name?

Reply:
I decided to move onto `GenerateName` which will have the same effect here, and will give a unique name for the machine - https://github.com/openshift/cluster-api-actuator-pkg/pull/160/files#diff-c7d1003b84db4b5057508a81ab203c46R80